lib/tasks/search.rake

```diff
@@ -1,170 +1,36 @@
 require 'task_helpers'
 
 namespace :search do
-  def import_from_cursor(cursor, index, opts)
-    tot = cursor.count
-    cnt = 0
-    t = Time.now
-    index.import cursor, {:method => :paginate, :per_page => opts[:batch_size]} do |documents|
```

Comment:
@clintonb: Please feel free to weigh in given you made the original decisions on this.  Thanks.